### PR TITLE
MER TX_Tb Report v2.3 

### PR DESCRIPTION
**Pull request by dbaluku**
Pending work on this report is 
- Values for the Indicator "The number of ART patients who were screened for TB at least once during the reporting period by Specimen sent"

### CODE REVIEW

**Comment by dbaluku** [comment]
@ssmusoke  please review

**Comment by slubwama** [comment]
@dbaluku  add Ticket link to the PR

**Comment by dbaluku** [comment]
https://app.asana.com/0/1116981488119523/1117011082450731/f

File: omod/src/main/resources/apps/ugandaemr_reports_app.json [hunk 1]
```diff
@@ -453,6 +453,14 @@
         "label": "HMIS 106a With DSDM Aggregations",
         "url": "reportingui/runReport.page?reportDefinition=c05879b8-77db-4a91-90a9-34631cd0ca3b",
         "order":99
+      },
+      {
```
Comment (ssmusoke) [inline omod/src/main/resources/apps/ugandaemr_reports_app.json]: @dbaluku This report has to be close to the other MER_Indicator reports

**Pull request merged**